### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by thewildjacko**
Finished Module 6 HW

### CODE REVIEW

File: README_HW6.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Part 1 - Short answer questions
+
+a) In SwiftUI, you describe your interfaces **declaratively** and leave the **rendering** to the Framework.
+
+b) When a View first appears, and you want it to animate in, you can do so in the Views **`.onAppear`** modifier.
+
+c) In Swift, the native types **Int** and **String** implement the Hashable protocol
+
+d) **Source** is how long an animation takes to complete.
```
Comment (byaruhaf) [inline README_HW6.md:9]: Duration not Sources

File: README_HW6.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+h) **`ScrollViewReader`** allows you to scroll to any position inside a ScrollView programmatically.
+
+i) True or False: All animations in SwiftUI are interruptible and reversible by default. **True**
+
+j) The **`GridItem` array** defines how items in a Grid should be sized and aligned.
+
+k) Wrap any changes you want to animate in a call to  **`withAnimation()`**
+
+l) The native SwiftUI grid view builds on the **Stack** and **List** views.
```
Comment (byaruhaf) [inline README_HW6.md:25]: VStack & HStack

**Review by byaruhaf — APPROVED** [review]
Well done, Jake! You've truly gone above and beyond. Your work showcases excellent use of animation, TabView, Identifiable, and the Navigation toolbar. Keep up the fantastic effort!

File: README_HW6.md (hunk 1 above)
Comment (thewildjacko) [reply README_HW6.md:9]: Don't know where "Source" came from—I don't think I meant to type that!

Comment (byaruhaf) [reply README_HW6.md:9]: i figured

**Pull request merged**